modules/installation/snip-installing-the-operator-common-steps.adoc

```diff
@@ -0,0 +1,94 @@
+. Create the `rhdh-operator` namespace that will contain the {product} Operator:
++
+[source,terminal]
+----
+$ kubectl create namespace rhdh-operator
+----
+
+. Create a pull secret using your {company-name} credentials to pull the container images from the protected {company-name} Ecosystem:
++
+[source,terminal]
+----
+$ kubectl -n rhdh-operator create secret docker-registry rhdh-pull-secret \
+    --docker-server=registry.redhat.io \
+    --docker-username=<redhat_user_name> \
+    --docker-password=<redhat_password> \
+    --docker-email=<email>
+----
+
+. Create a catalog source that contains the {company-name} Ecosystem Operators:
++
+[source,terminal,subs="attributes+"]
+----
+$ cat <<EOF | kubectl -n rhdh-operator apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: redhat-catalog
+spec:
+  sourceType: grpc
+  image: registry.redhat.io/redhat/redhat-operator-index:v{ocp-version}
+  secrets:
+  - "rhdh-pull-secret"
+  displayName: {company-name} Operators
+EOF
+----
+
+. Create an operator group:
++
+[source,terminal]
+----
+$ cat <<EOF | kubectl apply -n rhdh-operator -f -
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: rhdh-operator-group
+EOF
+----
+
+. Create a subscription to install the {product} Operator:
++
+[source,terminal,subs="attributes+"]
+----
+$ cat <<EOF | kubectl apply -n rhdh-operator -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: rhdh
+  namespace: rhdh-operator
+spec:
+  channel: fast
+  installPlanApproval: Automatic
+  name: rhdh
+  source: redhat-catalog
+  sourceNamespace: rhdh-operator
+  startingCSV: rhdh-operator.v{product-bundle-version}
+EOF
+----
+
+. Wait until the Operator deployment finishes.
+Run the following command to verify that the {product} Operator is running:
++
+[source,terminal]
+----
+$ kubectl -n rhdh-operator get pods -w
+----
```

Comment:
`kubectl ... get pods -w` will wait indefintely and display the changes happening to the pod. So this command will never return unless it is stopped.
And actually, the Operator Deployment will never run as it needs the pull secret so that the image can be pulled from registry.redhat.io.
I think we just need to wait for the Deployment to be created, with a simple command like this:
```sh
until kubectl -n rhdh-operator get deployment rhdh-operator &>/dev/null; do
  echo -n .
  sleep 3
done
echo "RHDH Operator Deployment created"
```

Reply:
changed